Dockerfile

```diff
@@ -10,6 +10,8 @@ ENV DOCKER_BUILD="true"
 # NOTE: if you need to change this, change the $CERT_WEBROOT_PATH env
 WORKDIR /app
 
+RUN apk add --no-cache python3 make g++
```

Comment:
Wondering if we require these and if so, could you please a comment for why we require?

Reply:
This one is because I am running on arm on my Mac M1. Seems like that running it with `--platform linux/amd64` compiles. Do you want me to remove it or I leave it in case you want to compile it for arm64?

On the other hand, I've seen that those dependencies aren't needed in line 28, so I'll remove them in either case.

Reply:
Ah I see, thanks for pointing that out. Lets keep it and just add a comment in place, why we have it in here.